src/ai/backend/manager/reporters/hub.py

```diff
@@ -0,0 +1,67 @@
+import asyncio
+from dataclasses import dataclass
+
+from ai.backend.manager.reporters.base import (
+    AbstractReporter,
+    FinishedActionMessage,
+    StartedActionMessage,
+)
+
+
+@dataclass
+class ReporterHubArgs:
+    reporters: dict[str, list[AbstractReporter]]  # Key: action type, Value: reporter instance
```

Comment:
```suggestion
type ActionType = str
reporters: dict[ActionType, list[AbstractReporter]]
```

Reply:
For this version, I'll skip the type alias.